participant/app/routes/$localAgency/recertify/index.tsx

```diff
@@ -39,7 +41,7 @@ export default function Index() {
         {resetSession === "true" && (
           <Alert
             type="warning"
-            headingLevel="h6"
+            headingLevel="h2"
```

Comment:
I changed this for the axe accessibility alert about using the next step up in headings. It does not change the appearance visually.

Reply:
Axe has feelings!